docs/flowcollector-flows-netobserv-io-v1beta2.adoc

```diff
@@ -201,13 +205,15 @@ Otherwise it is matched as a case-sensitive string.
 | List of additional features to enable. They are all disabled by default. Enabling additional features might have performance impacts. Possible values are: +
 
 - `PacketDrop`: enable the packets drop flows logging feature. This feature requires mounting
-the kernel debug filesystem, so the eBPF pod has to run as privileged.
+the kernel debug filesystem, so the eBPF agent pods have to run as privileged.
 If the `spec.agent.ebpf.privileged` parameter is not set, an error is reported. +
 
 - `DNSTracking`: enable the DNS tracking feature. +
 
 - `FlowRTT`: enable flow latency (sRTT) extraction in the eBPF agent from TCP traffic. +
 
+- `NetworkEvents`: enable the network events monitoring feature, such as correlating flows and network policies. This feature requires mounting
+the kernel debug filesystem, so the eBPF agent pods have to run as privileged.
 
```

Comment:
@jotak @msherif1234 so should i just manually remove this from the product docs when I copy it over?

Reply:
yeah actually I can also do it right here

Reply:
(it's part of 1.7 but won't work yet with current openshift)